coala_quickstart/generation/SettingsFilling.py

```diff
@@ -165,3 +168,93 @@ def resolve_anomaly(setting_name,
                                                    join_names(values)),
                   REPORT_ANOMALY_COLOR))
     return input()
+
+
+def require_setting(setting_name, setting_info, section):
+    """
+    This method is responsible for prompting a user about a missing setting and
+    taking its value as input from the user.
+
+    :param setting_name: Name of the setting missing
+    :param setting_info: A dictionary of the form
+
+    ::
+        {
+            "help_text": "help string for the setting",
+            "bears": [SomeBear", "SomeOtherBear"],
+            "type": bool
+        }
+
+    :param section:      The section the action corresponds to.
+    :return:             The preferred value provided by the user
+                         for the setting.
+    """
+    needed = join_names(setting_info["bears"])
+
+    STR_GET_VAL_FOR_SETTING = ('\nPlease enter a value for the setting \"{}\" '
+                               '({}) needed by {} for section \"{}\": ')
```

Comment:
Instead of `"{}"`, use `{}` and `repr(...)` the var you're using there when you `.format` it